Add Bigtable Setup To Storm Deploy

waprin opened the pull request:
This is just a first pass that works for me, I am assuming that it would be better as some type of conditional but I wanted someone who knows the code better to take a look.

Commit:
Add Bigtable Setup To Storm Deploy

AngusDavis commented:
A few overall nits: 
When expanding variables, use double quotes to avoid spaces causing trouble (e.g., STORM_LIB_DIR="${STORM_INSTALL_DIR}/lib"). Storm is probably the least well maintained extension and so there are a few bad spots already, but I'd like to avoid increasing the debt.

Consider moving the ALPN JAR into a subdirectory (STORM_INSTALL_DIR/lib/google)

The conditional is probably a good idea. Maybe a ENABLE_STORM_BIGTABLE?

Do you have a sample that writes to CBT? I'm not certain we have a good spot to put it, I'm more curious for myself :)

Commit:
Makes Storm Bigtable Configurable

Also fixes some nits around escaping, moves
ALPN jar into {STORM_INSTALL}/lib/google

AngusDavis commented:
Closing in favor of #32.
